Directory.Build.targets

```diff
@@ -4,6 +4,8 @@
     <ReadMePath>$(MSBuildProjectDirectory)\README.md</ReadMePath>
     <ReadMeExists Condition="Exists('$(ReadMePath)')">true</ReadMeExists>
     <PackageReadmeFile Condition="'$(PackageReadmeFile)' == '' And '$(ReadMeExists)' == 'true'">README.md</PackageReadmeFile>
+
+    <PackageIconFullPath Condition="'$(SuppressAspireIcon)' != 'true'">$(SharedDir)Aspire_icon_256.png</PackageIconFullPath>
```

Comment:
This is the only change that I wasn't absolutely sure if we wanted. @mitchdenny can you confirm if we want this or not in main?

Reply:
We don't want this change, and it just broke https://github.com/dotnet/aspire/pull/817 again.